Mkdocs deployment + styling (#18)
datajoint/element-facemap: docs/src/tutorials.md

# Tutorials

## Installation

> Installation of the Element requires an integrated development environment and database.
> Instructions to setup each of the components can be found on the
> [User Instructions](datajoint.com/docs/elements/user-instructions) page. 

Reviewer: ```suggestion
Installation of the Element requires an integrated development environment and database.
Instructions to setup each of the components can be found on the
[User Instructions](https://datajoint.com/docs/elements/user-guide) page. 
```

There, you'll find instructions referencing other DataJoint Element 'Workflows' (e.g.,

> [workflow for Element DeepLabCut](https://github.com/datajoint/workflow-deeplabcut)),

Reviewer 2: ```suggestion
[workflow for Element Facemap](https://github.com/datajoint/workflow-facemap)),
```

Author: Because facemap doesn't have notebooks, I considered it still 'under development' and I'm suggesting DLC as an example. If we put together the notebooks, I would change this

Reviewer: Yes, we don't currently have a `workflow-facemap`.

which serve as models for developing custom pipelines. While the workflow for Facemap is

> still under development, these can serve as a model for integrating across Elements.

Reviewer: ```suggestion
which serves as a model for developing custom pipelines. While the workflow for Facemap is
still under development, these can serve as a model for integrating Elements into a pipeline.
```

> ## Additional Materials
> 
> Coming soon!

Reviewer: ```suggestion
```
